VMBackup/main/config/extension.logrotate

```diff
@@ -0,0 +1,23 @@
+/var/log/azure/Microsoft.Azure.RecoveryServices.VMSnapshotLinux/extension.log {
+    # Old versions of log files are compressed with gzip by default.
+    compress
```

Comment:
if gzip util is missing in the VM then what is the expected handling?

Reply:
gzip is unix system cmd. So it should be present. Also logrotate is being used by other agents, so we should be safe. In worst case it won't do logrotate and silently fail without exceptions.